src/entry.js

```diff
@@ -1,6 +1,20 @@
+import React from 'react'
 import ReactDOM from 'react-dom'
+import { Provider } from 'react-redux'
+
 import routes from './routes'
+import store from './store'
+
 import '../sass/app.scss'
 
-const div = document.getElementById('app')
-ReactDOM.render(routes, div)
+const rootEl = document.getElementById('app')
+const render = () => {
```

Comment:
nevermind, i'm wrong!

```
const render = () => ReactDOM.render(<Foo />, rootEl)

render()
store.subscribe(render)
```

https://github.com/reactjs/redux/blob/master/examples/counter/src/index.js

can we just simplify the store.subscribe line like above?